Gadget.Notifications/Domain/Enums/NotifierType.cs

```diff
@@ -1,8 +1,8 @@
 ﻿namespace Gadget.Notifications.Domain.Enums
 {
     public enum NotifierType
-    {
-        Discord,
-        Email
+    {   None=0,
```

Comment:
nie musisz pisac x=0, w enumie bedzie to defaultowo numerowac od 0..n

Reply:
już poprawiam

Reply:
ale często tak piszę, bo jak ktoś zmieni kolejność pól to może się coś popsuć.